### PR TITLE
Cache improvements

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull request cache some jinja snippets.

### CODE REVIEW

File: udata_gouvfr/theme/templates/footer.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{% cache 60*5, 'footer', g.lang_code|default(config.DEFAULT_LANGUAGE) %}
```
Comment (davidbgk) [inline udata_gouvfr/theme/templates/footer.html]: No `default` for the other ones?

Comment (noirbizarre) [inline udata_gouvfr/theme/templates/footer.html]: No, it's just for the snippet used in error pages.
Another alternative is to set the value for error pages.

Comment (davidbgk) [inline udata_gouvfr/theme/templates/footer.html]: Ah, got it. Fair enough.

**Comment by davidbgk** [comment]
r+

**Pull request merged**